packages/ui/src/common/components/buttons/FilterButtons.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { ButtonGhost, ButtonPrimary } from '@/common/components/buttons'
+
+interface FilterButtonsProps {
+  onClear?: () => void
+  onApply?: () => void
+}
```

Comment:
I think that it might be too early to introduce this component. That's because it is the only one we have for now.

Usually we should extract common parts when we see two or more occurrences of duplicated code. Up-front optimizations might not be needed.

However, if you see that this particular one will be used more let me know.

Reply:
I thought so but then I saw that these buttons are now used on the select filters in the new design [@figma](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=3034%3A2041). This is why I extracted this into a component.